src/lib/y2partitioner/sequences/add_partition.rb

```diff
@@ -31,6 +33,7 @@ def initialize(disk_name)
         textdomain "storage"
         @disk_name = disk_name
         @ptemplate = PartitionTemplate.new
+        @options = FormatMount::Options.new
```

Comment:
I do not thing that it is independent, but lets see it when we start implementing other parts of options.